org.eclipse.scout.rt.client/src/main/java/org/eclipse/scout/rt/client/ui/basic/table/TableTileGridMediator.java

```diff
@@ -89,6 +89,22 @@ public void tableChanged(TableEvent e) {
           tileMappings.forEach(tm -> tm.getTile().dispose());
           setTileMappings(new ArrayList<>());
           break;
+        case TableEvent.TYPE_ROW_FILTER_CHANGED:
+          List<ITableRow> filteredRows = m_table.getRows().stream()
+              // Analogous to JsonTable.preprocessBufferedEvents() do not consider user filtered rows here, since these are handled directly by ui.
+              .filter(row -> row.isFilterAccepted() || row.isRejectedByUser())
+              .collect(Collectors.toList());
+
+          // delete tiles which are not accepted by filter
+          Predicate<ITableRowTileMapping> filteredOutPredicate = m -> !filteredRows.contains(m.getTableRow());
+          tileMappings.stream().filter(filteredOutPredicate).forEach(m -> m.getTile().dispose());
+          tileMappings.removeIf(filteredOutPredicate);
+
+          // insert non-existing tiles
+          filteredRows.removeAll(tileMappings.stream().map(ITableRowTileMapping::getTableRow).collect(Collectors.toList()));
+          tileMappings.addAll(m_table.createTiles(filterTopLevelTableRows(filteredRows)));
+
+          setTileMappings(tileMappings);
```

Comment:
Although this is the last case in the switch, add an explicit `break` at the end of this block. It does not matter now, but if in the future someone adds another case, they might easily miss adding the necessary break, causing unexpected behavior.